bundles/edu.kit.kastel.sdq.eclipse.grading.view/src/edu/kit/kastel/eclipse/grading/view/controllers/AssessmentViewController.java

```diff
@@ -105,54 +88,16 @@ public void addAssessmentAnnotation(IMistakeType mistake, String customMessage,
 			 * setAttribute(...)) without getting a too messy code
 			 */
 			e.printStackTrace();
-			this.alertObserver.error("Unable to create marker for annotation", e);
+			this.getAlertObserver().error("Unable to create marker for annotation", e);
 		}
 
 	}
-	
-	/**
-	 * Formats a custom penalty message. It will always use the message of the mistake, however iff the provided customMessage
-	 * is not null, it will append a \n and this custom message.
-	 * @param mistake the mistake to load the message from
-	 * @param customMessage the custom message to append (can be null)
-	 * @return the formatted message
-	 */
-	private String formatCustomPenaltyMessage(IMistakeType mistake, String customMessage) {
-		if (customMessage != null) {
-			return mistake.getMessage() + "\n" + customMessage;
-		} else {
-			return mistake.getMessage();
-		}
-	}
 
 	/**
 	 * creates markers for current annotations in the backend
-	 * The method will prevent duplicated markers resulting from 
-	 * having them in the local project and on the server.
 	 */
 	public void createAnnotationsMarkers() {
-		this.getAnnotations().stream().filter(annotation -> !isAnnotationPresent(annotation)).forEach(this::createMarkerForAnnotation);
-	}
-	
-	/**
-	 * Checks whether the given annotation is present in the currently opened project 
-	 * (An annotation is identified by its UUID)
-	 * @param annotation the annotation to check
-	 * @return true if the annotation is present, false if not
-	 */
-	private boolean isAnnotationPresent(IAnnotation annotation) {
-		try {
-			IMarker[] markers = AssessmentUtilities.getFile(annotation.getClassFilePath(), this.systemwideController.getCurrentProjectName()).findMarkers(null, false, 100);
-			for (IMarker marker : markers) {
-				if (annotation.getUUID().equals(marker.getAttribute(AssessmentUtilities.MARKER_ATTRIBUTE_ANNOTATION_ID))) {
-					return true;
-				}
-			}
-			return false;
-		} catch (CoreException e) {
-			// If the project (or file) can not be loaded the annotation is definitely not present 
-			return false;
-		}
```

Comment:
Please revert. Please have a look at this file. the changes may be a result of a merge commit. But this file should not be changed